nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/Run/UpdateHandlers/GroupUpdateAllVersionsHandler.cs

```diff
@@ -146,19 +139,29 @@ private async Task RunGroupedDependencyUpdates(Job job, DirectoryInfo originalRe
 
             if (updateOperationsPerformed.Count > 0)
             {
-                var commitMessage = PullRequestTextGenerator.GetPullRequestCommitMessage(job, [.. updateOperationsPerformed], group.Name);
-                var prTitle = PullRequestTextGenerator.GetPullRequestTitle(job, [.. updateOperationsPerformed], group.Name);
-                var prBody = await PullRequestTextGenerator.GetPullRequestBodyAsync(job, [.. updateOperationsPerformed], [.. updatedDependencies], experimentsManager);
-                await apiHandler.CreatePullRequest(new CreatePullRequest()
+                var existingPullRequest = job.GetExistingPullRequestForDependencies(
```

Comment:
Is there any need to call ClosePullRequest if the previous PR is made up of the same dependencies, but earlier versions?  Or is that already addressed elsewhere?

Reply:
Good question.  Closing a PR is treated the same as updating a PR and is handled in `RefreshGroupUpdatePullRequestHandler.cs`.  I verified the high level behavior with the NPM updater and now both emit the same events given the same job shape.

Reply:
In your scenario specifically, a new PR will be created, but the old one left alone.  Only if `job.is-updating-a-pull-request` is `true` will `close_pull_request` be called.  Both NPM and NuGet updaters do the same thing (with this change.)